### PR TITLE
Replace ioredis with Mock

### PR DESCRIPTION
**Pull request by jedtan**

### CODE REVIEW

File: graphql/__mocks__/ioredis.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+/* eslint-env jest */
+const ioredisMock = jest.requireActual('ioredis-mock')
```
Comment (kmjennison) [inline graphql/__mocks__/ioredis.js:2]: I'm a little confused by the purpose of this. The `requireActual` is getting the underlying module and then exporting it again, which should have no effect. Do you mean to create a mock from the module?

**Review by kmjennison — CHANGES_REQUESTED** [review]

File: graphql/__mocks__/ioredis.js (hunk 1 above)
Comment (jedtan) [reply graphql/__mocks__/ioredis.js:2]: From how I understand having this as `ioredis.js` means that the ioredis package is being mocked, and I'm basically replacing it's full implementation with `ioredis-mock` instead.

Comment (kmjennison) [reply graphql/__mocks__/ioredis.js:2]: Forgive my poor skimming here. I managed to miss the `-mock`! 🙄

**Review by kmjennison — COMMENTED** [review]
LGTM! ☕

File: graphql/__mocks__/ioredis.js (hunk 1 above)
Comment (jedtan) [reply graphql/__mocks__/ioredis.js:2]: No worries!

**Pull request merged**